test/Mechanical/rotational.jl

```diff
@@ -228,6 +228,37 @@ end
     # Plots.scatter(sol[friction.w], sol[friction.tau], label="")
 end
 
+@testset "lossy gear" begin
+    @mtkmodel LossyGear begin
+        @components begin
+            drive = Blocks.Sine(frequency=1, amplitude=10)
```

Comment:
RN, this throws error. Splitting it to `using Blocks: Sine` and then `drive = Sine(frequency=1, amplitude=10)` should work now.

- Needs https://github.com/SciML/ModelingToolkit.jl/pull/2330 for Blocks.Sine to work inside the `@mtkmodel`